crates/generator/src/verification/transaction.rs

```diff
@@ -93,6 +99,14 @@ impl<'a, S: State + CodeStore> TransactionVerifier<'a, S> {
                 )
                 .into());
             }
+            // Native token transfer
+            if p.is_native_transfer() {
+                // Verify to_id is CREATOR_ID
+                let to_id = raw_tx.to_id().unpack();
+                if Some(to_id) != self.polyjuice_creator_id && self.polyjuice_creator_id.is_some() {
```

Comment:
if ```polyjuice_creator_id``` is ```None```, this check will pass? Caller must ensure ```polyjuice_creator_id``` is ```Some```?
Maybe pass ```beckend_manage``` and call ```utils::get_polyjuice_creator_id``` if ```polyjuice_creator_id``` is ```None```?